Client/Frontend/Browser/PasswordHelper.swift

```diff
@@ -30,7 +30,7 @@ class PasswordHelper: BrowserHelper {
         if let path = NSBundle.mainBundle().pathForResource("PasswordHelper", ofType: "js") {
             if let source = NSString(contentsOfFile: path, encoding: NSUTF8StringEncoding, error: nil) as? String {
                 var userScript = WKUserScript(source: source, injectionTime: WKUserScriptInjectionTime.AtDocumentEnd, forMainFrameOnly: true)
-                browser.webView.configuration.userContentController.addUserScript(userScript)
+                browser.configuration.userContentController.addUserScript(userScript)
```

Comment:
Careful, you're setting the `configuration` of the `browser`, not the `webView`. As described in the [WKWebView docs](https://developer.apple.com/library/ios/documentation/WebKit/Reference/WKWebView_Ref/#//apple_ref/occ/instm/WKWebView/initWithFrame:configuration:), the `configuration` object is used only to initialize the webView object, after which it's "dead". If the webView is already created by the time this helper is initialized, this line will have no effect.

I recommend making `configuration` private as mentioned above to help us avoid this.